test/onnx/test_pytorch_onnx_caffe2_quantized.py

```diff
@@ -204,5 +204,19 @@ def forward(self, x, y):
         y = np.random.rand(1, 4, 3, 4).astype("float32")
         self.generic_test(QConcatModule(), (x, y,), input_names=["x", "y"])
 
+    def test_max_pool2d(self):
+        class QMaxPool2dModule(torch.nn.Module):
+            def __init__(self):
+                super(QMaxPool2dModule, self).__init__()
+                self.quant1 = torch.quantization.QuantStub()
+                self.dequant = torch.quantization.DeQuantStub()
+
+            def forward(self, x):
+                res = torch.nn.functional.max_pool2d(self.quant1(x), kernel_size=2, stride=1, padding=0)
+                return self.dequant(res)
+
+        x = np.random.rand(1, 2, 8, 8).astype("float32")
+        self.generic_test(QMaxPool2dModule(), (x,), input_names=["x"], decimal=0)
```

Comment:
We should be using a larger value for decimal, from the docs:
https://docs.scipy.org/doc/numpy-1.15.0/reference/generated/numpy.testing.assert_almost_equal.html
abs(desired-actual) < 1.5 * 10**(-decimal)

Can we use decimal of 5?